### DISCUSSION TITLE
tab player screen

### DISCUSSION THREAD

**Discussion by 2004pril** [ciderapp/Cider#769] [Q&A]
I bought it from the window store. If I enlarge the player screen, it will run on the full screen, but can't I make the play screen bigger on the window(tab)?
And the part where the playerbar doesn't work properly TT
https://user-images.githubusercontent.com/77568338/161070447-011ef2b6-b458-4ca8-a10f-c4711b2a02e0.mp4

**Comment by KillerParticles** [comment]
I don't really understand what you are trying to do;

> it will run on the full screen, but can't I make the play screen bigger on the window(tab)?
are you trying to make the window full screen? You do that by the buttoms up the top right. If you mean the album artwork make that bigger, try using another theme.

Some more info could help me understand.

**Reply by Monochromish** [reply]
They are talking about Cider not having a windowed fullscreen player and forcing full screen.

**Reply by KillerParticles** [reply]
Oh that? From what I recall that's been a thing since the start of Cider, I never took it for a bug. Thought that was intended.

**Reply by 2004pril** [reply]
> They are talking about Cider not having a windowed fullscreen player and forcing full screen.

thats what I want to say.. its a bit disappointing if intentional design..;(

**Reply by cryptofyre** [reply]
hit F11 when in the fullscreen mode, it’ll go windowed mode as a temporary solution. We’ll look into making the window’d type mode better and more accessible *and draggable.*

**Reply by 2004pril** [reply]
> 전체 화면 모드에서 F11 키를 누르면 임시 솔루션으로 창 모드로 전환됩니다. 우리는 window'd type 모드를 더 좋고 접근하기 쉽고 끌기 쉽게 만드는 방법을 조사할 것 _입니다._

thx!! :)
